lib/order.rb

```diff
@@ -0,0 +1,40 @@
+require_relative 'menu'
+#require_relative 'make_call'
+
+
+class Order
+
+attr_reader :order, :phone_number, :order, :cust_order
```

Comment:
You've defined `:order` twice here and it isn't used anywhere.  Nevertheless, it would lead to `order.order` which doesn't make sense.  A method with the same name as the class is an indication of a design problem.